.gitignore

```diff
@@ -0,0 +1,9 @@
+﻿################################################################################
```

Comment:
I've been using Visual Studio's automatically generated gitignore. It makes it pretty easy to customize and add or remove files from it. Do you think I should use the one from the gitignore repo instead?